templates/index.html

```diff
@@ -15,7 +15,7 @@
 	{% include 'header.html' %}
 	<title>{{ board.url }} - {{ board.title|e }}</title>
 </head>
-<body class="8chan {% if mod %}is-moderator{% else %}is-not-moderator{% endif %}" data-stylesheet="{% if config.default_stylesheet.1 != '' and not mod %}{{ config.default_stylesheet.1 }}{% else %}default{% endif %}">
+<body class="8chan {% if mod %}is-moderator{% else %}is-not-moderator{% endif %}" data-stylesheet="{% if config.default_stylesheet.1 != '' and not mod %}{{ config.default_stylesheet.1 }}{% else %}default{% endif %} active-{% if not no_post_form %}index{% else %}ukko{% endif %}">
```

Comment:
IIRC I added that for stylesheet switching JS logic.

On 2015-09-26 11:18 pm, Marcin Łabanowski wrote:
> In templates/index.html [1]:
> 
>> @@ -15,7 +15,7 @@
>> {% include 'header.html' %}
>> <title>{{ board.url }} - {{ board.title|e }}</title>
>> </head>
>> -<body class="8chan {% if mod %}is-moderator{% else
> %}is-not-moderator{% endif %}" data-stylesheet="{% if
> config.default_stylesheet.1 != '' and not mod %}{{
> config.default_stylesheet.1 }}{% else %}default{% endif %}">
>> +<body class="8chan {% if mod %}is-moderator{% else
> %}is-not-moderator{% endif %}" data-stylesheet="{% if
> config.default_stylesheet.1 != '' and not mod %}{{
> config.default_stylesheet.1 }}{% else %}default{% endif %} active-{%
> if not no_post_form %}index{% else %}ukko{% endif %}">
> 
> @data-stylesheet ?
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/ctrlcctrlv/infinity/pull/529#discussion_r40494459
> [2] https://github.com/ctrlcctrlv/infinity/pull/529/files#r40494459